.mdlrc

```diff
@@ -0,0 +1 @@
+rules "~MD013", "~MD024"
```

Comment:
Are these MDL rules Chef standards?

Reply:
There are no standards, except for 013 (long lines), it is really a judgement call by the person writing it.